osl_python_template_cli.py

```diff
@@ -0,0 +1,189 @@
+import argparse
+
+import inquirer
+from colorama import Fore, Style
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        prog="osl-python-template",
+        description="CLI tool for calling cookieninja",
+    )
+    parser.add_argument(
+        "--delete",
+        action="store_true",
+        help="Delete and re-download the template",
+    )
+    return parser.parse_args()
+
+
+def prompt_questions():
+    questions = [
+        inquirer.Confirm(
```

Comment:
we probably could just download it every time the CLI is called .. instead of asking that.
also this path looks from my own machine "/home/xmn" XD
so, please just remove the "delete" option. thanks!